docs: update to make styling more consistent (#34)
asyncapi/design-system: README.md

[![AsyncAPI Design System](/assets/github-repobanner-designsystem.png)](https://www.asyncapi.com)

>   # :deciduous_tree:Environment Setup

Author: Looks colourful I guess

>   # :deciduous_tree:Environment Setup

Reviewer: Ok, so please remove that empty spaces `  ` at the beginning of header.

>   # :deciduous_tree:Environment Setup

Author: Can I create a new pull request as I deleted my fork repository mistakenly instead of deleting my branch?

>   # :deciduous_tree:Environment Setup

Author: I have made the new pull request.

### :joystick:How to setup storybook locally

- Before starting the Storybook environment locally, install all the required dependencies. Run the command below to install the dependencies:

```cmd

> npm install --save-dev

Reviewer: Hmmm 🤔  It should be normal `npm install`.

Author: Ok.

```

> - Now, as you have all the `node_modules` installed in your project, you can start the Storybook environment. Run the command below to start it:

Reviewer 2: ```suggestion
- Now, that you have all the `node_modules` installed in your project, you can start the Storybook environment with the following command:
```

Reviewer 2: I know that this was here before, but the grammar/sentencing is a bit weird so I updated it to read a little bit better
